environments/staging-aws-test/meta.yml

```diff
@@ -0,0 +1,7 @@
+deploy_env: staging
+env_monitoring_id: staging-aws-test
+users:
+#  - dimagi
+#  - dimagi-qa
+  - droberts
```

Comment:
`droberts` seems like a value you were using for testing. Is that still necessary?

Reply:
I'll probably remove soon, but it's still useful for now.